### PR TITLE
Add Sbpayment::Link::RoutineEntry

### PR DESCRIPTION
**Pull request by banyan**
This is the endpoint for D01-1 and the name "RoutineEntry"  is taken from sbps specification.

### CODE REVIEW

File: lib/sbpayment/link/routine_entry.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require_relative '../parameter_definition'
+
+module Sbpayment
+  module Link
+    # D01-1
```
Comment (kachick) [inline lib/sbpayment/link/routine_entry.rb]: :memo: `定期申込要求`

**Comment by kachick** [comment]
:+1:

**Comment by kachick** [comment]
Would you release?  `0.5.3` or `0.6.0`, both versions as LGTM.

**Comment by banyan** [comment]
ok, I'll release! Thanks!

**Comment by kachick** [comment]
:bow:

**Pull request merged**